packages/api/endpoints/granules.js

```diff
@@ -877,16 +1051,23 @@ async function bulkDelete(req, res) {
     },
   };
 
-  log.debug(`About to invoke lambda to start async operation ${asyncOperationId}`);
-  await startAsyncOperation.invokeStartAsyncOperationLambda(asyncOperationEvent);
+  log.debug(
+    `About to invoke lambda to start async operation ${asyncOperationId}`
+  );
+  await startAsyncOperation.invokeStartAsyncOperationLambda(
+    asyncOperationEvent
+  );
   return res.status(202).send({ id: asyncOperationId });
 }
 
 async function bulkReingest(req, res) {
   const payload = req.body;
-  const numOfGranules = (payload.query && payload.query.size)
```

Comment:
This auto-linting doesn't look right, but perhaps it's a prettier default? 

Not this PR but we should probably be opinonated here and set a sensible lint default (https://eslint.org/docs/latest/rules/operator-linebreak)

Reply:
https://github.com/nasa/cumulus/pull/3225/commits/ea12a2846db17b78e9107a5109a445018d7ddfc8